### ISSUE TITLE
Agros Suite

### ISSUE THREAD

**Issue by marcfehling** [dealii/publication-list#349]
I've found the "Agros Suite", which is using deal.II and Hermes as third party libraries.

As they are using the library, we should also put (part of) their work into the publication list. We need to check their publications and filter in which they use/refer to deal.II

http://www.agros2d.org/about/

**Comment by marcfehling** [comment]
http://www.agros2d.org/automatic-hp-adaptivity/ :-)

**Comment by bangerth** [comment]
Yes, indeed, agros uses deal.II. We should indeed use their citations for ours as well where applicable.
